src/Consolonia.Core/Drawing/DrawingContextImpl.cs

```diff
@@ -473,31 +487,31 @@ private void FillRectangleWithBrush(IBrush brush, IPen pen, Rect r)
             double width = r2.Width + (pen?.Thickness ?? 0);
             double height = r2.Height + (pen?.Thickness ?? 0);
             for (int x = 0; x < width; x++)
-            for (int y = 0; y < height; y++)
-            {
-                int px = (int)(r2.TopLeft.X + x);
-                int py = (int)(r2.TopLeft.Y + y);
-                Color backgroundColor = brush.FromPosition(x, y, (int)width, (int)height);
-
-                CurrentClip.ExecuteWithClipping(new Point(px, py), () =>
+                for (int y = 0; y < height; y++)
```

Comment:
Probably this formatting adjustments wil be reverted by automatic cleanup anyway. We need to change appropriate editor config settings. But I'm not sure